dd-sdk-android/src/test/kotlin/com/datadog/android/log/internal/logger/DatadogLogHandlerTest.kt

```diff
@@ -107,6 +113,16 @@ internal class DatadogLogHandlerTest {
 
     @BeforeEach
     fun `set up`(forge: Forge) {
+        // To avoid java.lang.NoClassDefFoundError: android/hardware/display/DisplayManagerGlobal.
+        // This class is only available in a real android JVM at runtime and not in a JUnit env.
+        Choreographer::class.java.setStaticValue(
```

Comment:
maybe we can use capabilities of `TestConfigurationExtension` for that?

Reply:
It doesn't work :(. @xgouchet tried also before and I also tried in this PR, for some reason it crashes when you want to change that static final variable through reflection due to a weird state of the JVM. It only works if you call it here and it doesn't work actually to get the original one and put it back after. It also crashes with the same `IllegalModifier` exception.